pkg/cmd/pr/shared/editable.go

```diff
@@ -45,6 +45,12 @@ type EditableProjects struct {
 	ProjectItems map[string]string
 }
 
+func (s *EditableSlice) ReplaceValue(v []string) {
+	s.Value = v
+	s.Add = set.NewFromSlice(v).Difference(set.NewFromSlice(s.Default)).ToSlice()
+	s.Remove = set.NewFromSlice(s.Default).Difference(set.NewFromSlice(v)).ToSlice()
```

Comment:
Should these sets by reused?

```suggestion
	vSet := set.NewFromSlice(v)
	defaultSet := set.NewFromSlice(s.Default)
	s.Add = vSet.Difference(defaultSet).ToSlice()
	s.Remove = defaultSet.Difference(vSet).ToSlice()
```